examples_tests/__init__.py

```diff
@@ -162,7 +162,7 @@ def install_snap(self, example_dir, snap_name, version):
     def assert_command_in_snappy_testbed(self, command, expected_output):
         if not config.get('skip-install', False):
             output = self.run_command_in_snappy_testbed(command)
-            self.assertEqual(output, expected_output)
+            self.assertEqual(expected_output, output)
```

Comment:
Ah okay. Yeah I wish unittest would show you expected/actual-- I always have to look at the test itself to determine it.